web-registry/src/lib/db/types/json-ld/project-metadata.ts

```diff
@@ -1,69 +1,82 @@
-import { TypeValue, UrlList, UrlType } from 'lib/rdf/types';
+import { NameUrl, TypeValue, UrlList, UrlType } from 'lib/rdf/types';
 
+/** Anchored metadata AKA "Additional Info" - Editable only with a signed Ledger TX. */
 export interface ProjectMetadataLD {
   '@context': Context;
-  '@type': string;
+  '@type': string; // ex: regen:C01-Project
   'schema:name': string;
-  'schema:image': UrlType;
-  'schema:creditText': string;
-  'schema:description'?: string;
   'schema:location': any;
   'regen:projectType': string;
-  'regen:projectActivity': ProjectActivity;
-  'regen:projectDeveloper': ProjectStakeholder;
-  'regen:landSteward': ProjectStakeholder;
-  'regen:landOwner': ProjectStakeholder;
-  'regen:projectOriginator': ProjectStakeholder;
+  'regen:projectActivity': NameUrl;
   'regen:offsetGenerationMethod': string;
-  // TODO: not sure why regen:offsetGenerationMethod isn't compacting as expected so added this:
+  // regen:offsetGenerationMethod doesn't get compacted for project metadata because
+  // in COMPACTED_CONTEXT, we specify: 'regen:offsetGenerationMethod': { '@container': '@list' }
+  // while in project metadata, it's a string, not a list
+  // this context is used in credit classes metadata, we should probably update the credit classes metadata
+  // to use offsetGenerationMethods (plural!) since it can be a list
```

Comment:
@wgwz I spotted some inconsistencies in our projects/credit classes metadata which is causing some issue on the front-end when trying to compact the metadata
we always use `regen:offsetGenerationMethod` singular even when there can be multiple in credit class metadata (C01 has "Emission Reductions" and "Carbon Removals" for instance)

cc/ @S4mmyb